seed-data-scripts/seed.js

```diff
@@ -0,0 +1,362 @@
+const axios = require('axios');
+const fs = require('fs');
+const cuid = require('cuid');
+const titleIDStore = new Map();
+// Manually create the GOD problem via seed data per env then update the DOI here
+titleIDStore.set(
+    'What makes everything we can detect in the universe around us the way that it is, and why?', // GOD problem text
+    'why' // GOD problem DOI
+);
+
+const godProblemTitle = 'What makes everything we can detect in the universe around us the way that it is, and why?';
+
+const tsv = fs.readFileSync('./initialdata.txt', 'utf-8');
+const loadData = async () => {
+    const rows = tsv.split('\n');
+    let index = -1;
+
+    for (const row of rows) {
+        index++;
+
+        try {
+            // Skip header of csv
+            if (index === 0) {
+                continue;
+            }
+
+            const splitRow = row.split('\t');
+            let [title, parent1, parent2, content] = splitRow;
```

Comment:
This seems to be very specific to this set of data. Can't there be more than 2 parents? I wonder if this script will work for other similar sets of data

Reply:
currently that's the format Alex is supplying in, we won't be running this script again for some time after these sets of data